api/management/commands/acquiredata.py

```diff
@@ -0,0 +1,29 @@
+import os
+from urllib.request import urlretrieve
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
+            dest='path',
+            default='data',
+            help='Path to location of data files.',
+        )
+
+    def handle(self, *args, **options):
```

Comment:
I removed `*args` and it does not appear to break anything. Though I was worried that some Django function calling `handle` might pass in positional arguments. But it seems fine... will remove.